includes/class-signature.php

```diff
@@ -206,7 +206,10 @@ public static function maybe_double_knock( $response, $parsed_args, $url ) {
 		// Remove this filter to prevent infinite recursion.
 		\remove_filter( 'http_response', array( self::class, 'maybe_double_knock' ) );
 
-		if ( 401 === wp_remote_retrieve_response_code( $response ) ) {
+		$response_code = \wp_remote_retrieve_response_code( $response );
+
+		// Fall back to Draft Cavage signature for any 4xx responses.
+		if ( 400 <= $response_code && $response_code < 500 ) {
```

Comment:
@pfefferle Are we being to broad with this? So far I've only observed 401s and 400s. And a 500 for Ghost, but I think that may be a one-off.

Reply:
I think I am fine with checking all 4xx codes for now. Otherwise we could limit to 400 and 401 and start logging error codes!?